### ISSUE TITLE
Insufficient Contrast.

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#484]
# Tabs

Visibility: 2
Evaluation: Partially Supports
Success Criteria: 1.4.3: Contrast (Minimum)
Needs design: 

### Description
The tabs, such as those on the Messages sent and Templates pages, fail to meet the minimum criteria with only 1.7:1 contrast ratio when the active styles are applied to an unselected tab. 

![](https://dl.airtable.com/.attachments/aa3261825e59ef7e2bd0fb65dc0c301f/5758b279/Capturedecranle2022-03-15a13_19_22.png)

**Comment by yaelberger-commits** [comment]
Needs to be merged
